indexers/base.go

```diff
@@ -16,10 +16,10 @@ import (
 )
 
 // indexes a document
-const indexCommand = `{ "index": { "_id": %d, "_type": "_doc", "version": %d, "version_type": "external", "routing": %d} }`
+const indexCommand = `{ "index": { "_id": %d, "version": %d, "version_type": "external", "routing": %d} }`
 
 // deletes a document
-const deleteCommand = `{ "delete" : { "_id": %d, "_type": "_doc", "version": %d, "version_type": "external", "routing": %d} }`
```

Comment:
Note to self: need to check if removing this still works against indexes created with type